configure

```diff
@@ -25685,33 +25685,39 @@ fi
 
 for module in $ac_shared_modules ; do
   moduledir=`echo "$module" | sed -e 's/\.la$//'`;
+  modulename=`echo "$module" | sed -e 's/\.o$//'`;
```

Comment:
Here we define a new variable, `$modulename`, which is the module without the `.o` suffix, as it would have from the source list.